src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
-body {
-  margin: 0;
+$padding: 10px;
+$gap: 1px;
+$columns: 7;
+$cellSize: 100px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  padding: $padding;
+  max-width: ($cellSize + $gap) * ($columns + 1);
+  margin: 0 auto;
```

Comment:
The calendar should be centered both horizontally and vertically. Simply add flex to body (no need for margin here)